tools/src/h5diff/ph5diff_main.c

```diff
@@ -63,7 +63,7 @@ main(int argc, char *argv[])
     MPI_Comm_size(MPI_COMM_WORLD, &g_nTasks);
 
     if (g_nTasks == 1) {
-        printf("Only 1 task available...doing serial diff\n");
+        fprintf(stderr, "Only 1 task available...doing serial diff\n");
```

Comment:
This output statement messes makes it so the same output diff reference files can't be used between h5diff and ph5diff testing. Move this print statement to stderr since it doesn't seem that important anyway.